packages/components/psammead-navigation/src/index.jsx

```diff
@@ -33,22 +39,25 @@ const StyledUnorderedList = styled.ul`
   padding: 0;
   margin: 0;
   position: relative;
-  overflow: hidden;
+
+  ${({ isSwipeable }) => !isSwipeable && `overflow: hidden;`}
```

Comment:
As `isSwipeable` is set currently set by javascript checking the window's width, could we replace this with a media query here? Then we will get the correct navigation on the server render as well?

Reply:
Good spot 👍